src/callbacks_step/analysis_surface_2d.jl

```diff
@@ -0,0 +1,116 @@
+using Trixi
+using Trixi: integrate_via_indices, norm, apply_jacobian_parabolic!, @threaded,
+             indices2direction,
+             index_to_start_step_2d, get_normal_direction, dot, get_node_coords
+import Trixi: analyze, pretty_form_ascii, pretty_form_utf
+
+struct AnalysisSurfaceIntegral{Indices, Variable}
+    indices::Indices
+    variable::Variable
+end
+
+struct ForceState{RealT <: Real}
+    Ψl::Tuple{RealT, RealT}
+    rhoinf::RealT
+    uinf::RealT
+    linf::RealT
+end
+
+# TODO - This should be a struct in ForceState
+struct FreeStreamVariables{RealT <: Real}
+    rhoinf::RealT
+    uinf::RealT
+    linf::RealT
+end
+
+struct LiftForcePressure{RealT <: Real}
+    force_state::ForceState{RealT}
+end
+
+struct DragForcePressure{RealT <: Real}
+    force_state::ForceState{RealT}
+end
+
+function LiftForcePressure(aoa::Real, rhoinf::Real, uinf::Real, linf::Real)
+    Ψl = (-sin(aoa), cos(aoa))
```

Comment:
```suggestion
    # Ψl is the normal unit vector to the freestream direction
    Ψl = (-sin(aoa), cos(aoa))
```

Reply:
I just saw this (and I know it's an old comment), but for Trixi.jl we follow the convention of using only ASCII characters for executable code, see https://trixi-framework.github.io/Trixi.jl/stable/styleguide/#Conventions

Reply:
I see, thanks!